vms/platformvm/service.go

```diff
@@ -2170,10 +2205,7 @@ func (s *Service) IssueTx(req *http.Request, args *api.FormattedTx, response *ap
 		return fmt.Errorf("couldn't parse tx: %w", err)
 	}
 
-	s.vm.ctx.Lock.Lock()
-	defer s.vm.ctx.Lock.Unlock()
-
-	if err := s.vm.Network.IssueTx(req.Context(), tx); err != nil {
+	if err := s.vm.issueTx(req.Context(), tx); err != nil {
```

Comment:
This function doesn't seem to honor the documented invariant on `issueTx` (by calling `tx.ID()` after `issueTx` returns).

Reply:
Ah I was wondering why we created this variable in the X-Chain implementations, that makes sense.

Reply:
I think this is safe to do now that we're only initializing once from https://github.com/ava-labs/avalanchego/pull/2538